CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Unreleased
+
+- Next.js middleware function `isAuthenticated` fails more loudly; previously it
+  returned false in the case of a Convex backend that didn't expose an endpoint
+  called `auth:isAuthenticated`, now it throws an error. This should help people
+  with the migration required for 0.0.76.
+
 ## 0.0.78
```

Comment:
Technically this is unreleased too, I think - I just speculatively added the next version # when I updated this file earlier.

Reply:
ah gotcha